File: ContactsList.Server/Model/Person.cs
using ContactsList.Server.Model;
using Microsoft.AspNetCore.Identity;

public class ApplicationUser : IdentityUser
{
    public string Name { get; set; }
    public string Surname { get; set; }
    public int CategoryId { get; set; }
    public Category Category { get; set; }
    public int? SubcategoryId { get; set; }
    public Subcategory Subcategory { get; set; }
    public string MobilePhoneNumber { get; set; }
    public DateTime BirthDate { get; set; }
}

File: contactslist.client/src/shared/models/person.ts
import { Category } from "./category";
import { Subcategory } from "./subcategory";

export class Person {
    Id: String = ''
    name: string = ''
    surname: string = ''
    email: string = ''
    categoryId: number | null = null
    category: Category | null = null
    subcategoryId?: number | null = null
    subcategory?: Subcategory | null = null
    birthDate: Date = new Date()
    PasswordHash: string = ''
}